### DISCUSSION TITLE
PP and rank are not shown in my profile

### DISCUSSION THREAD

**Discussion by isuewo** [ppy/osu#21831] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/53264315/209717091-37710255-9b75-4f9f-a2bb-1252f1a95afe.png)
https://lazer.ppy.sh/users/29724850

Best Performance(s) on my profile are also empty (possibly related to #21830).

I assume this is because I only played Lazer and never logged into Stable? Also, I use the AppImage, so that could maybe also be the cause.

**Comment by peppy** [comment, marked as answer]
Could you please try to play again? It should in theory fix your profile when you do.

**Reply by isuewo** [reply]
![image](https://user-images.githubusercontent.com/53264315/211511652-0d32186d-b30d-45b5-ac5a-b2dff8a79f2e.png)

That seems to have fixed the pp count and my rank. Thank you! 
My ranked score and the score ranks are still at 0 though.

**Reply by peppy** [reply]
Yep, those are expected. Will be fixed in the future.

**Reply by isuewo** [reply]
Alright, looking forward to that. Thanks again.

**Reply by Fearknight134** [reply]
I umm have the exact same problem
![image](https://user-images.githubusercontent.com/128107946/228691725-8673d72d-fb44-48ed-856f-7920a7a14723.png)

**Reply by Fearknight134** [reply]
and your solution is not working..
